.eslintrc.cjs

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  ...require('@linzjs/style/.eslintrc.js'),
```

Comment:
Where did this format come from, the linz standard for js files is `'` following prettier's default?

Reply:
I don't know - I just ran `pre-commit run --all-files`. I don't have any dotfiles in my root directory which look relevant.

Reply:
See https://github.com/linz/stac/pull/34/checks?check_run_id=3647362020